schema.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE surveys(
+   id serial PRIMARY KEY,
+   name text NOT NULL
+ );
+                                                        
+ CREATE TABLE questions(
+  id serial PRIMARY KEY,
+  survey_id int REFERENCES surveys(id) ON DELETE CASCADE NOT NULL,
+  text text NOT NULL
```

Comment:
I totally see what you're saying with 'questions.question' not being ideal syntax, however, personally I still think it'd be more readable and clear if it was question over text.